profiles/build.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	deploymentYAML = "deploy/operator.yaml"
+	configMapName  = "default-profiles"
+	profilesDir    = "profiles"
+)
+
+type matcher struct {
+	foundConfigMapType bool
+	foundConfigMapName bool
+	foundData          bool
+	replace            bool
+}
+
+func main() {
+	file, err := os.Open(deploymentYAML)
```

Comment:
I might take a pass after this is merged and working to see if we can make this a more general purpose tool for converting any seccomp profiles to a `ConfigMap`

Reply:
Yeah that would be cool, thank you! :pray: